Python/Product/Analysis/OverloadResult.cs

```diff
@@ -267,6 +262,20 @@ internal BuiltinFunctionOverloadResult(PythonAnalyzer state, string name, IPytho
             : this(state, name, overload, removedParams, null, extraParams) {
         }
 
+        private static IEnumerable<string> GetInstanceDescriptions(PythonAnalyzer state, IEnumerable<IPythonType> type) {
+            foreach (var t in type) {
+                var av = state.GetAnalysisValueFromObjects(t);
+                var inst = av?.GetInstanceType();
+                if (inst.IsUnknown()) {
```

Comment:
Does this take care of https://github.com/Microsoft/PTVS/issues/4279 ?

Reply:
No, this only applies to the return types. But it'll likely be a similar change in the same class for that one.